org.lflang/src/org/lflang/ASTUtils.xtend

```diff
@@ -177,7 +177,7 @@ class ASTUtils {
      * if the port on the left or right of the connection involves a bank of reactors or a multiport.
      * @param connection The connection.
      */
-    private static def boolean isWide(Connection connection) {
+    static def boolean isMultiportConnection(Connection connection) {
```

Comment:
The name isMultiportConnection for this method seems misleading because the connection may not involve any multiports and the method could still return true.

Reply:
I found `isWide` also not very clear, because it could be multiple individual connections in case of `o1, o2 -> i1, i2`. Do you have a suggestion?

Reply:
How about isMultipleConnection?

Reply:
I renamed it to `hasMultipleConnections`, which seems better from a grammatical point of view.